srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableEntry.java

```diff
@@ -167,7 +180,22 @@ public String getMeasureDate(final String key)
      */
     public void setMeasureDate(final String key, final String retrievalString)
     {
-        fDynamicValues.put(key + VariableEntry.SEPARATOR + VariableEntry.RETRIEVAL_STRING, retrievalString);
+        fDynamicValues.put(makeRetrievalInfoKey(key), retrievalString);
+    }
+    
+    /**
+     * Puts the given retrieved value into the dynamic values map under the given key,
+     * also populating the human-readable retrieval info.
+     * @param key the input key to populate
+     * @param retrievedValue the retrieved value
+     * @return {@code this} for convenience
```

Comment:
I looked at the JDK's [ProcessBuilder](http://docs.oracle.com/javase/7/docs/api/java/lang/ProcessBuilder.html) for reference. They all say "Returns this process builder." I'll update mine accordingly.